appender.go

```diff
@@ -0,0 +1,157 @@
+package duckdb
+
+/*
+#include <duckdb.h>
+*/
+import "C"
+
+import (
+	"database/sql/driver"
+	"errors"
+	"fmt"
+	"time"
+	"unsafe"
+)
+
+type appender struct {
```

Comment:
Wouldn't it make sense to make the `appender` exported? Then it would also appear in the documentation.